tests/e2e/api/routing.js

```diff
@@ -445,12 +445,12 @@ describe('routing', () => {
         results.forEach((result, idx) => {
           if (idx === 0) {
             // online user request
-            expect(result.limit).toEqual(1);
-            expect(result.fields).toEqual('all_fields');
+            expect(result.limit).to.equal(1);
+            expect(result.fields).to.equal('all_fields');
           } else {
             // offline user requests
-            expect(result.statusCode).toEqual(403);
-            expect(result.responseBody.error).toEqual('forbidden');
+            expect(result.statusCode).to.equal(403);
```

Comment:
I think it would add value if we can use some library which has enums/constants defining a list of http status codes versus hard coding them here?